assets/scss/components/_modal.scss

```diff
@@ -5,8 +5,7 @@ $modal-width: 440px;
 	@include transition(transform .2s ease);
 	@include centerAbsolute($modal-width);
 	min-height: 0;
-	width: $modal-width;
-	margin-top: 10vh;
+	width: $modal-width !important;
```

Comment:
It does for the sake of storybook. The only downside is that custom modal widths will also need `!important`, but it doesn't seem like a big problem.